src/pages/TeamsLanding.js

```diff
@@ -3,8 +3,11 @@ import TeamsList from "../components/teams/TeamsList";
 
 const TeamsLanding = () => {
```

Comment:
the component does not provide any logic,
maybe you can merge it into <TeamsList /> ?